### PR TITLE
add new user attributes registered_as_guest and registered_under_domain

### PR DESCRIPTION
**Pull request by Amtron**

### CODE REVIEW

File: typescript/src/entities/user.ts [hunk 1]
```diff
@@ -53,6 +53,12 @@ export class User extends Entity {
   @User.property()
   public userType?: UserType;
 
+  @User.property()
+  public registeredAsGuest?: boolean;
+
+  @User.property()
+  public registeredUnderDomain?: number | null;
```
Comment (s3341458) [inline typescript/src/entities/user.ts:60]: this type a domain rather than number

File: python/users.py [hunk 2]
```diff
@@ -63,6 +63,8 @@ class User(sdk.python.entities.Entity):
     enable_store_notifications = Property(bool)
     is_super_user = Property(bool)
     is_admin_of_subscribed_company = Property(bool)
+    registered_as_guest = Property(bool)
+    registered_under_domain = Property(int)
```
Comment (s3341458) [inline python/users.py:67]: this one should be a domain rather than number

Comment (Unitix) [reply python/users.py]: I didn't want to add another relationship to the user. I only really needed the id ; making it a relationship is superfluous.

**Review by Unitix — APPROVED** [review]
LGTM

**Pull request merged**